pyccel/codegen/printing/codeprinter.py

```diff
@@ -60,7 +53,23 @@ def doprint(self, expr, assign_to=None):
         # Format the output
         return ''.join(self._format_code(lines))
 
+    def _print(self, expr):
+        """Annotates the AST.
+
+        The annotation is done by finding the appropriate function _visit_X
+        for the object expr. X is the type of the object expr. If this function
+        does not exist then the method resolution order is used to search for
+        other compatible _visit_X functions. If none are found then an error is
+        raised
+        """
```

Comment:
Was this docstring copied from the `_visit` method in the semantic parser? Please update it

Reply:
Guilty as charged. Fixed in e29d193